matter/wifi/rs911x/rsi_if.c

```diff
@@ -45,76 +45,80 @@ bool hasNotifiedWifiConnectivity = false;
  */
 static uint8_t wfx_rsi_drv_buf[WFX_RSI_BUF_SZ];
 wfx_wifi_scan_ext_t *temp_reset;
+uint8_t security;
 /*
  * Getting the AP details
  */
-int32_t wfx_rsi_get_ap_info(wfx_wifi_scan_result_t *ap)
-{
+int32_t wfx_rsi_get_ap_info(wfx_wifi_scan_result_t *ap) {
   int32_t status;
   uint8_t rssi;
-  ap->security = wfx_rsi.sec.security;
-  ap->chan     = wfx_rsi.ap_chan;
-  memcpy(&ap->bssid[0], &wfx_rsi.ap_mac.octet[0], 6);
+  ap->security = security;
+  ap->chan = wfx_rsi.ap_chan;
+  memcpy(&ap->bssid[0], &wfx_rsi.ap_mac.octet[0], COPY_6_CHAR);
```

Comment:
Why do we copy only 6 Char. The naming should examplin that
`BSSID_MAX_STR_LEN` would maybe clearer
Also how come do we copy the mac octs to bssid? is this normal?

Reply:
changed